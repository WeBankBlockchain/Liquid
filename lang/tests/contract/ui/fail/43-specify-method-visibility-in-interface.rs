use liquid_lang as liquid;

#[liquid::interface(name = auto)]
mod foo {
    extern "liquid" {
        pub fn bar() -> u32;
    }
}

fn main() {}
