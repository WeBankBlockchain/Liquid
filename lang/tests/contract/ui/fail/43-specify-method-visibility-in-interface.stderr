error: visibility modifier is not allowed for method declarations in interface
 --> $DIR/43-specify-method-visibility-in-interface.rs:6:9
  |
6 |         pub fn bar() -> u32;
  |         ^^^
